CodeExamples.md

```diff
@@ -55,32 +55,21 @@ mkdir marketplace-quickstart
 cd marketplace-quickstart
 npm init
 # Answer the questions in the command line prompt
-cat > marketplace.js
+touch marketplace.ts
```

Comment:
Again, we need to inform the user about how to compile and run the code if we are advising them to use typescript

Reply:
I mentioned that above too, amongst a variety of other problems.
```
There is nothing in the documentation that tells what the user needs to compile from .ts => .js, and run it. If the user tries to use node marketplace.js he will receive ES6 import-related errors.
```

**I believe the meta is the issue here is that this bit of code, is trying to become a generic exporter that goes from .ts => .md** Having a pipeline that goes from test.ts +> tutorial.md, is neat, but seems like a complete distraction.

**My recommendation:**
Forget building the script that generates the markdown. Just update the markdown/tutorial as needed so it can remain functional.

Reply:
In retrospect, it would have been easier if the test was written in js rather than ts as that just adds an additional step. The PR has improved a lot since you did your review above though so I don't think we should abandon it at this point